src/modules/about/index.js

```diff
@@ -0,0 +1,17 @@
+import Module from '../../abc/Module';
+
+import AboutScreen from './AboutScreen';
+console.log("hello from AboutModule");
+export default class AboutModule extends Module {
+
+	constructor(app, name) {
+		super(app, "AboutModule");
+
+		app.Router.addRoute({
+			path: '/auth/about',
```

Comment:
You should change path from `/auth/about` to `/about` since it's independent module.